pxd.c

```diff
@@ -146,6 +146,32 @@ static long pxd_ioctl_init(struct file *file, void __user *argp)
 	return pxd_read_init(&ctx->fc, &iter);
 }
 
+static long pxd_ioctl_resize(struct file *file, void __user *argp)
+{
+	struct pxd_context *ctx = NULL;
+	struct pxd_update_size_out update_args;
+	long ret = 0;
+
+	if (copy_from_user(&update_args, argp, sizeof(update_args))) {
+		return -EFAULT;
+	}
+
+	if (!ctx || ctx->id >= pxd_num_contexts_exported) {
```

Comment:
This condition is incorrect, ctx is initialized only below

Reply:
merge error, fixed now.